cdc/model/sink.go

```diff
@@ -107,6 +107,15 @@ type RowChangedEvent struct {
 	Keys         []string           `json:"keys"`
 }
 
+//GetColumnTypes get column types from dml query
+func (e *RowChangedEvent) GetColumnTypes() []byte {
+	res := make([]byte, 0, len(e.Columns))
```

Comment:
Maybe there are many columns with same type, will cause many duplicate elements
1. return `map[byte]struct{}` 
2. use `map[byte]struct{}`  filter type first, then convert it to `[]byte`.